apps/server/src/types/index.ts

```
@@ -0,0 +1,44 @@
+}
+
+export interface PaginationResponseType<T extends InitialPaginationResponseType> {
+  list: T["list"];
+  count: T["count"];
+  totalResult: number;
+  totalPage: number;
+  search?: string;
+  page: number;
+}
+
+type ERROR = { result: false; code: number; data: string };
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ERROR'. Consider to delete it or rename it.

'ERROR' is defined here:


(lint/suspicious/noRedeclare)